src/functions/revokeHandler.ts

```diff
@@ -5,13 +5,36 @@ import {
 } from "aws-lambda";
 import { logger } from "../logging/logger";
 import { LogMessage } from "../logging/LogMessage";
+import { getConfig } from "../config/getConfig";
+import { putObject } from "../common/aws/s3";
+import { createToken, StatusList } from "../common/token/createToken";
+
+const REQUIRED_ENV_VARS = [
+  "STATUS_LIST_BUCKET_NAME",
+  "SIGNING_KEY_ID",
+] as const;
 
 export async function handler(
   event: APIGatewayProxyEvent,
   context: Context,
 ): Promise<APIGatewayProxyResult> {
   logger.addContext(context);
   logger.info(LogMessage.REVOKE_LAMBDA_STARTED);
+
+  const config = getConfig(process.env, REQUIRED_ENV_VARS);
+
+  const requestJWT = event.body;
+  const uri = getRequestBody(requestJWT).uri;
+  const index = getRequestBody(requestJWT).idx;
```

Comment:
```suggestion
  const { body } = event;
  const { uri, index } = extractUriAndIndex(body)
```
I find `getRequestBody` a bit misleading because we already have the request body - it's `event.body`.

Reply:
Also, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring - I find this quite nice. :)